x-pack/solutions/security/plugins/security_solution/public/onboarding/config.ts

```diff
@@ -28,5 +29,6 @@ export const onboardingConfig: TopicConfig[] = [
     }),
     body: siemMigrationsBodyConfig,
     disabledExperimentalFlagRequired: 'siemMigrationsDisabled',
+    capabilitiesRequired: `${SECURITY_FEATURE_ID}.advancedInsights`,
```

Comment:
What does SIEM rule migrations have to do with `advancedInsights`? Isn't this an "Entity Analytics" capability?
Can we use `detections` instead?

Reply:
Fixed [here](https://github.com/elastic/kibana/pull/219427)

Reply:
Thank you @logeekal @semd